cicecore/cicedynB/general/ice_forcing.F90

```diff
@@ -5431,12 +5449,16 @@ subroutine box2001_data_ocn
          do j = 1, ny_block   
          do i = 1, nx_block   
 
+         this_block = get_block(blocks_ice(iblk),iblk)
+         iglob = this_block%i_glob
+         jglob = this_block%j_glob
+
          ! ocean current
          ! constant in time, could be initialized in ice_flux.F90
-         uocn(i,j,iblk) =  p2*real(j-nghost, kind=dbl_kind) &
-                            / real(nx_global,kind=dbl_kind) - p1
-         vocn(i,j,iblk) = -p2*real(i-nghost, kind=dbl_kind) &
-                            / real(ny_global,kind=dbl_kind) + p1
+         uocn(i,j,iblk) =  p2*real(jglob(j), kind=dbl_kind) &
+                            / real(ny_global,kind=dbl_kind) - p1
+         vocn(i,j,iblk) = -p2*real(iglob(i), kind=dbl_kind) &
+                            / real(nx_global,kind=dbl_kind) + p1
```

Comment:
This change is BFB on square domains -- but definitely fixing a bug in the denominators, thanks for catching it.

Reply:
Yes, I also noticed the bug in the forcing, not just when I was reviewing the paper.  So it wasn't working on multiple blocks and the forcing was incorrect.